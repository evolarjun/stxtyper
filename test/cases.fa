>PD-4797_multirow gi|424771548|ref|NZ_AMXQ01000137.1| Escherichia coli O111:H8 str. CFSAN001632 CFSAN001632_96, whole genome shotgun sequence
AACAAAAAACAACCAGATTCGACACTAAAAATTTTTATTTTTCTATATATCAACAACTTACACTGGTGGT
GATGGTGCCATAAAAATCAAAAAATGCGCCTTTTTCCGCGCCGCCCGCCCCGTGTTCAGGCCCACCCCAC
CAGGAGGACCCGCAAAATGATAATGGTTATCATTTGTAATGTAGTCCGGTTTCTTCCACCATCGCACCGG
ACCAGCGACTATGAGGGGACAACGCCGCGCTCCGTTAACGCGGTAAACCCCGGTGTGTATCGTTTTTGAT
TATCCCCGCACACTCGCGCAGAGGAGTCTCCCTGTCGGGCTGCGGTCTCTGTTAATGAGGGAATACAGCG
ACGATACGGCGCATCCGCAAAACTTAGTTCAGGCACTGAGTGCGGATATAGTCCTGTGCCCCTTCCAGCT
GCTTCTGCATTGTCATCAACCGTTCTCTGAGGATGAAATAATCCCGTTCAGCGGTGTCTGCCAGTCGGGG
GCCGGTTGCATTATCCACGCCGGAGGTGCCGGTGGCTTCACGCACGGTACCGGAGCAGGTGGCGTTGATC
CGCAGGCGCTTACGACCAGCGGCAACATCAGCACGCAGAGTTTCATTTTCAGCTCTCGCATCAGCTAATT
CCCTTGAGTATTTTGCATCGAGCGCAGCAACATCGCGCTGGCGCACCTGCATATCAGTAATGGTTGCGTT
TGCCAGCTCCAGCTCTCTGGCTTTTTTATCGCGCTGCGCTTTGTAGGTGATGGCGTTATCACGGTAATGA
TTCAGCCCCAGACTAAGCGCACCACAGGCCACCAGCAGGGCAATGATGACCACACACAGTACGCGGTTCA
TTTCACCACCAGCGTATCTGACCGATGAAATAACCGGAGACCATAATCACAAACACCAGCCAGATAAGAA
TGAACTTCCAGGTAGATAATTTTTCAGCCATCACTCGAATCTCCCGAATCAGTTTGCTAAAATCAAACAC
ACTTTCTCCTTTGACTTTTCCAGAGTCAGGAAACACAAAACCCCGCTTGCAGCCAACAAACGGGGTTTTT
ACTTTTATTCACTTAGTTTTTGTCAGTTCGCAGGATTTCGTGTTATCCGTCCGTGTGAGCAAACCGCATT
TTTCAGCAAAATATTCTGCTTATCTGTCAATTCCCCAGCACGCCAGCGCACTCTCCTGGTCGCGACGGGA
TACCTGACCGTAACAGTTGTTTGAGCGAATAGGCCGCGCTATTGCGCAGAGTGATTACTGTCGGATATTA
TTCGCCAGCTGAAATATTACTTCACGTTTTGTTGTTTATTCCTTGCCGCCAGCGTCTCCCTGCGCGGGCT
TTTTTTGTCCATAAGAAAGCCCCTCCGGAGAGGGGCTGGAGAGTGGCGCTATGTGCCATTGCATGGTGCC
GGGTGCCTCCCGGTGAGTTCGGCCTGGTGCCACCAAACCCGCGTATTCTCGCTTACGATCATCAAAGAGA
TCATACCGTTCACCAGTCGCCCCTCCGCACAGGGGGATTCACCATGCAGAAATTTTCTAACACATCTATT
ATCAGACCGGCAACAACTGACTGAATTGAGATGTATTTAACATTTATGAATCTCCGCCTGCTATTTTCAC
TGAGCTATTCTAAGTCAACGAAAAATAACTTCGCTGAATCCCCCTCCATTATGACAGGCATTAGTTTTAA
TGGTTACAGTCATCCCCGTAATTTGCGCACTGAGAAGAAGAGACTGAAGATTCCATCTGTTGGTAAATAA
TTCTTTATCACCCACTTTAACTGTAAAGGTATCGTCATCATTATATTTTGTATACTCCACCTTTCCAGTT
ACACAATCAGGCGTCGCCAGCGCACTTGCTGAAAAAAAATGAAAGCGATGCAGCTATTAATAATGTTTTT
TTCATTTTACCCCCTCAACTGCTAATAGTTCTGCGCATCAGAATTGCCCCCAGAGTGGATGAATCCCACA
ATATTTTATTGTGCGTAATCCCACGGACTCTTCCATCTGCCGGACACATAGAAGGAAACTCATCAGATGC
CATTCTGGCAACTCGCGATGCATGATGATGACAATTCAGTATTAATGCCACGCTTCCCAGAATTGCATTA
ATGCTTCCAAAAGAAATTCTTCCTACACGAACAGAGTCTTGTCCATGATAATCAGGCAGGACACTACTCA
ACCTTCCCCAGTTCAATGTAAGATCAACATCTTCAGCAGTCATTACATAAGAACGCCCACTGAGATCATC
CAGTGTTGTACGAAATCCCCTCTGTATTTGCCGAAAACGTAAAGCTTCAGCTGTCACAGTAACAAACCGT
AACATCGCTCTTGCCACAGACTGCGTCAGTGAGGTTCCACTATGCGACATTAAATCCAGATAAGAAGTAG
TCAACGAATGGCGATTTATCTGCATCCCCGTACGACTGATCCCTGCAACACGCTGTAACGTGGTATAGCT
ACTGTCACCAGACAATGTAACCGCTGTTGTACCTGGAAAGGTAACATGTGAAAAATCAGCAAAGCGATAA
AAAACATTATTTGTCCTGTTAACAAATCCTGTCACATATAAATTATTTCGTTCAACAATAAGCCGTAGAT
TATTAAACCGCCCTTCCTCTGGATCTATCCCTCTGACATCAACTGCAAACAAATTATCCCCTGTGCCACT
ATCAATCATCAGTAAAGACGTACCTCCTGATGAAATAGTCTGTAATGGAGTACCTATTGCAGAGCGAATG
ACATTCAGCGAATCTACATACGTCTTTGCAGTCGAGAAGTCTAAGGTAAATTCCTTCGCAACCACATTAA
CTGAAAAGATAACAAAGAAAAAAGTTAGCACTCTAAAAATAATTATTTTCATATTACACAATACTCCTTG
AGCACCATACGATAACTATATTCTTGACATCCTCACCCCCTGAAGGACGGCGTTTTACGGCGCACCGGAT
AAACGTAACAATAACGTAATGAAAATGATAATTATATTCAAAGAGAGCTGCAACCTTAACATATCTGGTC
AGATCTCATGCGACTACTTGACGTACGTAGACAACAACATTTATTGATACACAGGATGTTACGGACATAA
AAAAGCCAGCCACTGGGGGAGGCTGGCAAACTCGTAGAGCAAAATGCTGTTACGCAAACTTCGTTACAGG
GTCATCCTGCAATACAAAAAATACACAATATTTAGAAAACTAATAGTGCCATGTGCAATTTTTAAGATTT
TGTTATTAATTGTGGTCGCACCTTCCTTTCTGTGTACTTTCCGTATAGCTCACAGGATTCTGGGTACAAA
AAAACCCGCGCATCGGCGGGTTAAGCAGCGTGGCAATGTAACCACTCTTATCATGATATGCAGATTTTTA
CGATCGTAAACTATTTTTTCGCTGATAAAATACAGAGGTTCTCCCTCCCGGCAATTCACGCTCAACATAC
CGATCCATCTCAAGCCTCACTCCCAGCATCATCAGCATGCCTTCAACAATCCCCTCCGCTTTGTGAAGGC
GTTTACCTATACAGGTGTCAGAGCACCCATGTTTCCGTGCCAGCGCCATGAACGTCCCCCCCAGCACGTA
ATAATCAACCAGCAAGTCATGCAGATCGCGATTGTTCCGGTAAAGGCGGGCTATACACCCGCATATCACC
ATCGCATCATCGTCACAGCACTGCGGGCGTGATTTTACTTTTTCGGGGATCAGTCCCTTAAATCCGGCGG
CAATGGGTGACCAGGTCACATCCTCATGGTTATTTGCCGCCCATGCCCCCCAGCACTCAAGAACCTGCCG
GATATCACGCATCAGTATCTTTACCCCATCCGCGATGAACCATAAGAACACCGTTGACAATAGCGTGTCT
TTTTCCTTCTTTATCGCCAGTGTATTTTCTGACCGTGTTGCGACTACAGTTCAGTATTCTGGCTACCTCG
GTCTGATTTTCATATGCCTCAACGAGCATGTCAGGAATGGTTTTTACTGTGAACGTCATGCGGCCTCACT
TCTGCTGTTTCGCAGGTCTTTAAGTTTCTGCTGATACTTCGCCTTGATGGCCCTGCACTCTTCGACAGTC
CAGCGATGGCGGTTATGGTTTGATTCGATTTCGTCTACTGCTTCCTGCCCGATGCGATTAATCAGTTCGA
CGCGATACGGAACGAGATTTCCGCTTTTGTGCTGGTTGCACACCACGCATTGCTTGTGAATATTGCGTTC
ATCAAATCGGAGTTGAGGTGCCGCAGCAGTTGTCCGGTAATGTCCGGCATCCCACTGAGCAGACGTGAGC
GTTCCGCACGAGATACATGGTAAGTCGCGGTCTCTTTCTCTGATGAAGGCGTTTACGGCTTGTTGGGCTT
GTTTAATCCAGTAACTGCGGGGCTTTAAGGCGAGTTTTCGAATCTTAATTTTATCTTTCTGTTTCTGCTC
CTCTCGTCGTCGTTTCTTCTCTGCTGTTTTTTCCGCCTTTTCGCGTTCTTTATTTCGTCGTTCGAGTGCT
AATTTAGTTCCGTGTTCCGGGCTGCACCACCACTGATTTGAGAATGCCGGGTGAAACCATTCCTTACAGA
TTTTGCATTTCCTTCGCGCTGGTTTAGCCATTAAGCAGCCTCCCCTGTTACTTTAAGCATTCCGTTATCT
AGCAGCTTTCTTGTCAGCCACTGTTGACCACGCCCGGTGATTTTTGTGGTGAACGATATCTGTATTCCGT
GATTTGTATTGACCGCTGTTTCTTTCACTGTGAAATAGCCGCGATCCATATATTCCTGCATTGGCACATT
GCGCCGGGCACCTGAAGCAATAAGGATTTTGTGATCGCGCATCCACGCAAACCGTTTGTTTGGACCAATA
CCAACAACCTTTGCAAAGTTTCCAATCAAAATTCCGCTGGCCTCGCCAACTCGATCGGCAAACTCAACTT
TAGGTGCTGCGAGAGCAAGCTGTTTCTCCAGTTCAGCCTTCTGGTCTTCAAGGTCGGCCGCAAGGCGCAA
TGCCTCAGAAAAGGTTTGTGGTATTTTCGCGGTTGCCCCTTCGAGTTCTCGCCAGCGGTCAACAAGGCGA
GCGGTGAATTCCGGCGACAACTGGGCAACGACAATAATGCTATCGCGCTTACCTTGTTCGCCTTCGAAGA
CGTAAGCCTCGACACTACGGCGCAGCCCTAAGTTATTGATTTTTTTTCGAAAACCTGCAATGCAGGAAGT
TGAATAACTCCAGATTTAGCCAGGCGCTCTATTGATATTTTTACGTTATCTGGACGACTTCCCACTAACT
CAGCGATTTCAATGCTTGTCATTTTGATGATACTGCTATTTATCAGCTCATTCATTGTCATGTCCTCTCA
CATTGAAAATTCAGTAATAAAAAACCCAGCCGAAGCTGGGTTTGTTAAGTTGTCAATTGTCAGTAGCGAT
GCAGTGAAGGCGGCAACTCTTTGTTCTTAAGCCTTACCCATGCGGAAAGGTTCGTTGGTCCGTCTGGCTC
ATTAATATCAACATCTCGTGTGTGGTTGATTAAAACATCTCTCGCCATTCCGATAACATACGAAAACTCA
TGACCGTAGTCGTAGCATCTGCCGGAATAGTTCGATTGAATTTGTTTTAGCGCCGGATACAGTTCGCGGA
ATAATGCCTGTGAGCGGTTAGCATAATCCCATAACCATACAAGGCTGTTTGCTTCTTTTGCGGAAAGCTC
GTTGGTGCTCTTCTCTTGTTTGCCAGTATTTTTCTTGCACTGGCTGAAATAGCAGTCTTCCAGTTTTTCG
AACACCTCCCACGCCTGATCGGTTTCGAGCATTTTTGCATGACGGGCTGCGCCTCGTTCTGTCCAGAGGA
TGAGGGAGCGGGCTTTCGGGGAAATTTGTAACCCTCTTAAAGATGGTTGCAAATTTTGTGAGTAGTTTAA
AGCTACCCGCAAATTTTTCAGGGCATCGCCAGCCACTTTGAAAAAATGCTTTCCTTCAATAAAGCGCTCA
GCATTTCTGGCGTAGTTATTTTGAATGTTCTTAACTTCAGCACCGTACAACTGCGCCAAAAGTTCGGTGG
TAATAACAGGAATCTGGTTATGGGTGATCGGGGAGAGAGTTTCAACAGAGATTTGAGTTGTCATAATGAC
GCCCTCGAGTGGTTTCTTAATAACTCACCACCGACGACGCCAATCTTCTGGTGGTGAACTGTGCAGGGTT
GGCGTAACCGGGAAACCGACCGGCGCGGATCTCTCCGCCCCCACACAGCCCACCATAATTCAGATGTGCG
CGTGCATACGACAATAAAAAACACGCTCGCGGCGTGTATCTGTCACGGTCTCTATCCAGGACGCCAATCC
CGACGCCAGATTTTGCTGGCGCGTGAGGAATATAGCCCCGAATAAATCATCACGTCAATCACCTTGTTTT
CCTCGCACGATGTCTTAGCCACCGGATATCCCACAGGTGAGCCGTGTAATTGAAGGTTTTTACGTCAGAT
TCTTTTGGGATTGGCTTGCGTTTATTTCTGGAGCGTTTCGTTGGAAGGTATTTGCAGTTTTCGCAGATGA
TGTCGGTGATACTTCGTCGCTGTCGCCTCATGCAGCCCTCCTGACGCCCAGCCCGATCGCCATCAATGCC
GCTTTGGATACGGTAGTAAACATCCGTCGAGGACTGATGAACGGTCGCCAAATCAGCAACATTGAGCCTT
TGCTGTTTCCCTTCTTCTCCAGCCCTGTCGATGGTTCGATAAAATTAATCCGTCCATCAGTGATGATGCG
AACTTCGTCAACACTCTCCAGAGCCTTGCTGAACCATCCGACAGACATATCCTCTGGCACAAGCATCA

