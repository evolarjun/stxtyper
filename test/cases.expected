#target_contig	stx_type	operon	identity	target_start	target_stop	target_strand	A_reference	A_identity	A_coverage	B_reference	B_identity	B_coverage
PD-4797_multirow	stx1	PARTIAL	99.75	1625	2852	-	AAA71893.1	99.68	100.00	AAA71894.1	100.00	86.67
